.release-please-manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-  ".": "1.2.0",
-  "packages/api-explorer": "0.9.2",
-  "packages/extension-api-explorer": "21.4.1",
-  "packages/extension-sdk": "21.4.1",
-  "packages/extension-sdk-react": "21.4.1",
-  "packages/hackathon": "21.4.1",
-  "packages/run-it": "0.9.2",
+  ".": "1.3.0",
+  "packages/api-explorer": "0.9.3",
+  "packages/extension-api-explorer": "21.4.2",
+  "packages/extension-sdk": "21.5.0",
```

Comment:
should be 21.4.2

Reply:
Hm. Would have though extension-sdk was at 21.4.1. Only extension-sdk-react was impacted by yesterdays change

Reply:
correct: extension-sdk was at 21.4.1 and should be showing up as 21.4.2 in this PR which it is now